backslash/session.py

```diff
@@ -61,6 +61,10 @@ def report_test_start(self, name, file_name=NOTHING, class_name=NOTHING, test_lo
 
         return returned
 
+    def report_test_distributed(self, test_logical_id):
+        if 'report_test_distributed' in self.client.api.info().endpoints:
```

Comment:
@maormarcus this `if` should move to the plugin. If someone calls the method deliberately he/she won't expect it to do nothing silently...